src/components/icon/Icon.constants.ts

```diff
@@ -4,13 +4,16 @@ import { TIconGlyph, TIconSize } from './Icon.types';
 
 const DEFAULT_ICON_SIZE = 20;
 
-const ICON_SIZES: TIconSize[] = [10, 12, 16, 20, 28, 32, 40, 52, 64, 104];
+const ICON_SIZES: TIconSize[] = [10, 12, 14, 16, 18, 20, 24, 28, 32, 40, 52, 64, 104];
```

Comment:
I thought `14` and `18` are used in the `button` component (dont know where I got 24 from) and so added those to the array and type. It is kinda misleading in figma, since the icons dimensions are `18px x 18px` and the actual icon is only `16px x 16px` (size `16`)

Reply:
Yes, it's annoying that clicking on the icon selects the actual icon and not the container. Icons have been upsized for the most part so they fit better in the container.

As I suggested in another place, if there is a discrepancy between what we are seeing in practice vs what the docs say, we should double check with design before implementing something not in the docs.